addons/interaction/functions/fnc_passMagazineLocal.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+ * Author: BaerMitUmlaut
+ * Pass spare magazine for the specified weapon.
+ *
+ * Arguments:
+ * 0: Unit that passes the magazine <OBJECT>
+ * 1: Unit to pass the magazine to <OBJECT>
+ * 2: Magazine classname <STRING>
+ * 3: Amount of rounds in magazine <NUMBER>
+ * 
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_unit, _player, "30Rnd_65x39_caseless_mag", 30] call ace_interaction_fnc_magToPassazine
+ *
+ * Public: No
+ */
+
+#include "script_component.hpp"
+params ["_unit", "_target", "_magazine", "_ammoCount"];
+
+_target addMagazine [_magazine, _ammoCount];
+if (_target == ACE_player) then {
```

Comment:
Use the event version of the `displayStructuredText` and pass a target. It will only display if the target is `ACE_player`. This might be the best solution for displaying the hint.
```sqf
["displayTextStructured", [_target], ["string or format syntax", 1.5, _target]] call EFUNC(common,targetEvent);
```